pkg/requests/base.go

```diff
@@ -318,10 +318,6 @@ func compileRequestError(body []byte, statusCode int, isV2 bool) RequestError {
 	json.Unmarshal(body, &errorBody)
 
 	msg := "Request failed"
-	if statusCode == 401 && isV2 {
```

Comment:
This is an old, manual override for errors we used to provide before v2 had RAK support. Removed this since it's no longer accurate.